src/ng/directive/ngModel.js

```diff
@@ -1110,14 +1116,15 @@ var DEFAULT_REGEXP = /(\s+|^)default(\s+|$)/;
     <file name="index.html">
       <div ng-controller="ExampleController">
         <form name="userForm">
-          Name:
-          <input type="text" name="userName"
+          <label>Name:
+            <input type="text" name="userName"
                  ng-model="user.name"
                  ng-model-options="{ updateOn: 'blur' }"
-                 ng-keyup="cancel($event)" /><br />
-
-          Other data:
-          <input type="text" ng-model="user.data" /><br />
+                 ng-keyup="cancel($event)" />
+          </label><br />
```

Comment:
I've seen examples with both. I can certainly make them more consistent, but it's hard to tell which convention to use :)